pkg/host/kernel.go

```diff
@@ -36,12 +36,21 @@ type KernelInterface interface {
 	BindDpdkDriver(string, string) error
 	// BindDefaultDriver binds the virtual function to is default driver
 	BindDefaultDriver(string) error
+	// BindDriverByBusAndDevice binds device to the provided driver
+	// bus - the bus path in the sysfs, e.g. "pci" or "vdpa"
+	// device - the name of the device on the bus, e.g. 0000:85:1e.5 for PCI or vpda1 for VDPA
+	// driver - the name of the driver, e.g. vfio-pci or vhost_vdpa.
+	BindDriverByBusAndDevice(string, string, string) error
```

Comment:
Same situation here - the function doesn't use named args in the interface for consistency reasons because all other functions are defined without named args. I'm for adding named args for all methods at once in a separate PR

Reply:
I will propose to add named args to the interface as a part of https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/586

Reply:
for this specific function I added named args to the interface as a part of the PR